### PR TITLE
Fix: format comments

### PR DESCRIPTION
**Pull request by fengshunli**

### CODE REVIEW

**Comment by fengshunli** [comment]
@caoxianfei1 modify comment naming error

**Review by caoxianfei1 — COMMENTED** [review]
Delete blank line here and is not graceful. don't care if there are any rules.

File: pkg/chunkserver/volume.go [hunk 1]
```diff
@@ -16,8 +16,8 @@ const (
 
 // createFormatVolumeAndMount
 func (c *Cluster) createFormatVolumeAndMount(device curvev1.DevicesSpec) ([]v1.Volume, []v1.VolumeMount) {
-	vols := []v1.Volume{}
-	mounts := []v1.VolumeMount{}
+	var vols []v1.Volume
+	var mounts []v1.VolumeMount
```
[the remark this answers is not in the document]
Comment (fengshunli) [reply pkg/chunkserver/volume.go]: done

File: pkg/config/config.go [hunk 2]
```diff
@@ -11,18 +11,21 @@ const (
 
 const (
 	// configmap to record the endpoints of etcd
+
```
[the remark this answers is not in the document]
Comment (fengshunli) [reply pkg/config/config.go]: done @caoxianfei1 merge

**Review by caoxianfei1 — APPROVED** [review]
lgtm

**Review by caoxianfei1 — APPROVED** [review]
lgtm

**Pull request merged**